src/GraphQL/Validation/ValidationError.cs

```diff
@@ -42,7 +42,7 @@ public ValidationError(string originalQuery, string number, string message, para
             nodes?.Apply(n =>
             {
                 _nodes.Add(n);
-                if (n.SourceLocation != null)
+                if (n.SourceLocation != SourceLocation.Empty)
                 {
                     var location = new Location(new Source(originalQuery), n.SourceLocation.Start);
                     AddLocation(location.Line, location.Column);
```

Comment:
And here it adds the line/column to the error, without ever referencing the `SourceLocation.Line` or `SourceLocation.Column` property